doc/tutorials/platformer/app/lib/objects/star.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:ember_quest/ember_quest.dart';
+import 'package:flame/collisions.dart';
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flutter/material.dart';
+
+class Star extends SpriteComponent with HasGameRef<EmberQuestGame> {
+  late Vector2 _gridPosition;
+  late double _xPositionOffset;
+  Star({
+    required Vector2 gridPosition,
+    required double xPositionOffset,
+  }) : super(size: Vector2.all(64), anchor: Anchor.center) {
+    _gridPosition = gridPosition;
+    _xPositionOffset = xPositionOffset;
+  }
```

Comment:
Set these directly so that you don't have to use late, in the same way as on the others

Reply:
Fixed.